Stats/Supporting Files/nl.lproj/Localizable.strings

```diff
@@ -0,0 +1,212 @@
+//
+//  Localizable.strings
+//  Stats
+//
+//  Created by Phúc Ngô on 02/04/2020.
+//  Running on macOS Big Sur M1.
+//
+//  Copyright © 2020 Phúc Ngô. All rights reserved.
```

Comment:
Hi. Thanks for the translation. Could you please change the copyright to me?

Reply:
Hi @exelban Done.